src/css/main.css

```diff
@@ -8,7 +8,7 @@
    Base styles: opinionated defaults
    ========================================================================== */
 
-html {
+   html {
```

Comment:
This space should remove, is not an child tag or similar. @ciptohartanto Can you fix that on the formatting. Thx. 
All other looks good for me. Thanks